services/pull/check.go

```diff
@@ -69,12 +67,14 @@ func CheckPullMergable(stdCtx context.Context, doer *user_model.User, perm *acce
 		}
 
 		if err := pr.LoadIssue(ctx); err != nil {
+			log.Error("Unable to load issue[%d] for %-v: %v", pr.IssueID, pr, err)
```

Comment:
If we add this to every call of `LoadIssue`, then why not add it to `LoadIssue` directly?

Reply:
depends on where you want the logging to be referred to.